.changeset/tough-ears-study.md

```diff
@@ -0,0 +1,5 @@
+---
+'@celo/celocli': minor
+---
+
+Add ability to build and view governance proposals which interact with contracts verified on celoscan
```

Comment:
while the changes to make this work were not in this package its still nice to include a release note for users to read